mm2src/coins/eth.rs

```diff
@@ -2959,11 +2984,18 @@ impl EthCoin {
                 let fee_details: Option<EthTxFeeDetails> = match receipt {
                     Some(r) => {
                         let gas_used = r.gas_used.unwrap_or_default();
-                        let gas_price = web3_tx.gas_price.unwrap_or_default();
-                        // It's relatively safe to unwrap `EthTxFeeDetails::new` as it may fail
-                        // due to `u256_to_big_decimal` only.
-                        // Also TX history is not used by any GUI and has significant disadvantages.
-                        Some(EthTxFeeDetails::new(gas_used, gas_price, fee_coin).unwrap())
+                        let gas_price = web3_tx.gas_price.unwrap_or_default(); // TODO: create and use EthTxFeeDetails::from(web3_tx)
+                                                                               // It's relatively safe to unwrap `EthTxFeeDetails::new` as it may fail
+                                                                               // due to `u256_to_big_decimal` only.
+                                                                               // Also TX history is not used by any GUI and has significant disadvantages.
```

Comment:
Please move todo comment and old comment `It's relatively safe to unwrap EthTxFeeDetails::new as it may fail...` above the code (dont inline it).

Reply:
done [b21a186](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2051/commits/b21a18681471c4f59d1382d22edabc7b0cc3d186)